data_analysis/plots.py

```diff
@@ -405,12 +407,13 @@ def fetch_ridership_data() -> pd.DataFrame:
             available date.
```

Comment:
GH won't let me comment on the actual function definition, but I wonder if this `fetch_ridership_data` is also something that could be moved into a separate place, to try to just divide things by function a bit more?

Reply:
That's fine. I can move this to the a separate file along with the other calculations.